### PR TITLE
Move Weights and Fees changes and updates to the new repo

### PR DESCRIPTION
**Pull request by lsgunnlsgunn**

### CODE REVIEW

File: v3/docs/03-runtime/h-weights-and-fees/index.mdx [hunk 1]
```diff
@@ -381,30 +406,28 @@ impl<T: Get<Perquintill>> Convert<Fixed128, Fixed128> for TargetedFeeAdjustment<
 }
 ```
 
-## Next Steps
+## Next steps
 
 The entire logic of fees is encapsulated in `pallet-transaction-payment` via a `SignedExtension`.
```
Comment (sacha-l) [inline v3/docs/03-runtime/h-weights-and-fees/index.mdx:411]:
```suggestion
The entire logic of fees is encapsulated in `pallet-transaction-payment` via a [`SignedExtension`](/concepts/extrinsics#signed-extension).
```

File: v3/docs/03-runtime/h-weights-and-fees/index.mdx [hunk 2]
```diff
@@ -381,30 +406,28 @@ impl<T: Get<Perquintill>> Convert<Fixed128, Fixed128> for TargetedFeeAdjustment<
...
 While this module provides a high degree of flexibility, a user can opt to build their custom
```
Comment (sacha-l) [inline v3/docs/03-runtime/h-weights-and-fees/index.mdx:412]:
```suggestion
While this pallet provides a high degree of flexibility, a user can opt to build their custom
```

File: v3/docs/03-runtime/h-weights-and-fees/index.mdx [hunk 3]
```diff
@@ -381,30 +406,28 @@ impl<T: Get<Perquintill>> Convert<Fixed128, Fixed128> for TargetedFeeAdjustment<
...
 payment module drawing inspiration from Transaction Payment.
 
 Given now you know what Substrate weight system is, how it affect the transaction fee computation, and how to specify
```
Comment (sacha-l) [inline v3/docs/03-runtime/h-weights-and-fees/index.mdx:415]:
```suggestion
Given now you know what Substrate's weight system is, how it affects transaction fee computation, and how to specify 
```

File: v3/docs/03-runtime/h-weights-and-fees/index.mdx [hunk 4]
```diff
@@ -381,30 +406,28 @@ impl<T: Get<Perquintill>> Convert<Fixed128, Fixed128> for TargetedFeeAdjustment<
...
 ### Examples
 
-Substrate Recipes contains examples of both
-[custom weights](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/weights) and
-custom
-[WeightToFee](https://github.com/substrate-developer-hub/recipes/tree/master/runtimes/weight-fee-runtime).
+You can find examples of custom weights and fees in the following repositories:
+
+* [Custom weights](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/weights)
+* [custom WeightToFee](https://github.com/substrate-developer-hub/recipes/tree/master/runtimes/weight-fee-runtime).
```
Comment (sacha-l) [inline v3/docs/03-runtime/h-weights-and-fees/index.mdx:430]:
```suggestion
* [Custom WeightToFee how-to guide](/how-to-guides/v3/weights/calculate-fees)
```

**Review by sacha-l — APPROVED** [review]
Made a few edit suggestions. 🥇

File: v3/docs/03-runtime/h-weights-and-fees/index.mdx [hunk 5]
```diff
@@ -381,30 +406,28 @@ impl<T: Get<Perquintill>> Convert<Fixed128, Fixed128> for TargetedFeeAdjustment<
...
 
 ### Examples
 
-Substrate Recipes contains examples of both
-[custom weights](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/weights) and
-custom
-[WeightToFee](https://github.com/substrate-developer-hub/recipes/tree/master/runtimes/weight-fee-runtime).
+You can find examples of custom weights and fees in the following repositories:
+
+* [Custom weights](https://github.com/substrate-developer-hub/recipes/tree/master/pallets/weights)
```
Comment (sacha-l) [inline v3/docs/03-runtime/h-weights-and-fees/index.mdx:429]:
```suggestion
* [Custom weights for the Example pallet](https://github.com/paritytech/substrate/blob/master/frame/example/src/weights.rs)
```

**Review by lsgunnlsgunn — COMMENTED** [review]
applied suggestions

**Pull request merged**